odl/trafos/wavelet.py

```diff
@@ -571,6 +872,11 @@ def __init__(self, dom, nscales, wbasis, mode):
 
             'per': periodization -- like periodic-padding but gives the
             smallest possible number of decomposition coefficients.
+        axes : sequence of `int`, optional
+            Dimensions in which to calculate the wavelet transform.
+            The sequence's length has to be equal to dimension of the ``grid``
```

Comment:
This is not true. You can have lists of almost arbitrary lengths, as long as your input is large enough. The interpretation needs a bit more explanation here, i.e. what does it mean if an axis appears more than once?